projects/packages/forms/src/blocks/contact-form/components/jetpack-form-notifications-settings.js

```diff
@@ -0,0 +1,107 @@
+import { FormTokenField, ToggleControl } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import InspectorHint from '../../shared/components/inspector-hint';
+
+const JetpackFormNotificationsSettings = ( { setAttributes, notificationRecipients } ) => {
+	const [ localNotificationRecipients, setLocalNotificationRecipients ] =
+		useState( notificationRecipients );
+	const [ localFormNotifications, setLocalFormNotifications ] = useState(
+		notificationRecipients?.length > 0
+	);
+
+	// Fetch users who can edit posts (editors and admins)
+	const users = useSelect( select => {
+		const { getUsers } = select( coreStore );
+		return getUsers( { who: 'authors', per_page: -1 } ) || [];
+	}, [] );
+
+	// Filter to only include users with edit capabilities (editors and admins)
+	const eligibleUsers = users.filter( user => {
+		return user.capabilities?.edit_posts || user.capabilities?.edit_pages;
+	} );
+
+	// Create a map of user ID to user name for easy lookup
+	const userMap = {};
+	eligibleUsers.forEach( user => {
+		userMap[ user.id.toString() ] = user.name || user.slug;
+	} );
+
+	// Get the current post author
+	const { postAuthorId } = useSelect( select => {
+		const { getEditedPostAttribute } = select( editorStore );
+		const authorId = getEditedPostAttribute( 'author' );
+		return {
+			postAuthorId: authorId,
+		};
+	}, [] );
+
+	// Convert user IDs to names for display
+	const selectedUserNames = localNotificationRecipients
+		.filter( userId => userMap[ userId ] )
+		.map( userId => userMap[ userId ] );
+
+	// All available user names for suggestions
+	const allUserNames = eligibleUsers.map( user => user.name || user.slug );
+
+	return (
+		<>
+			<ToggleControl
+				label={ __( 'Enable form submission notifications', 'jetpack-forms' ) }
```

Comment:
Instead of form submissions can we use form responses for more consistency across UI?